mapping-for-navigation/_posts/0000-01-11-dual-carriageways.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Remodeling Dual-carriageways
+---
+
+![]({{site.baseurl}}/images/dual-carriageway/1200px-German_Autobahn_1936_1939.jpg)
+
+*A German dual carriageway in the 1930s ([Wikipedia](https://upload.wikimedia.org/wikipedia/commons/thumb/b/bc/German_Autobahn_1936_1939.jpg/1200px-German_Autobahn_1936_1939.jpg))*
+
+> A divided highway (also separated highway) is any highway where traffic flows are physically separated by a barrier (e.g., grass, concrete, steel), which prevents movements between said flows.
+>
+> [OpenStreetMap wiki]( https://wiki.openstreetmap.org/wiki/Editing_Standards_and_Conventions#Divided_highways)
+
+We oftentimes encounter cases where a bi-directional highway should be mapped as a dual-carriageway to better
+represent what's on the ground. OpenStreetMap mappers have adopted different approaches to mapping dual-carriageways
+based on the source of data, local community best practices and personal preference. This guide distills these best
+practices on how to improve road network data in OpenStreetMap by re-mapping bi-directional highways to dual-carriageways.
+
+Our basic rule of thumb are:
+
+*   Keep things simple, add complexity only when necessary.
+*   Dual-carriageway should have a physical barrier visible from imagery or in some cases road marking that show its illegal to pass on the other side (MUTCD).
+*   There is enough length from the physical divider (~20 meters?).
```

Comment:
@maning 20 meters, is it a length of the divider?

Reply:
Yes, but I don't feel comfortable adding this part.  As a best practice document, we should recommend when and when not to map a dual-carriageway, but the 20 m feels overly presriptive.

Reply:
Yes, I agree with you let's skip this value of 20 meters for now. Will describe this issue in a "How to trace roads" guide.